lib/collection/src/operations/verification/query.rs

```diff
@@ -105,6 +106,35 @@ impl StrictModeVerification for QueryGroupsRequestInternal {
 }
 
 impl StrictModeVerification for CollectionQueryRequest {
+    async fn check_custom(
+        &self,
+        collection: &Collection,
+        strict_mode_config: &StrictModeConfig,
+    ) -> CollectionResult<()> {
+        // check for unindexed fields in formula
+        if strict_mode_config.unindexed_filtering_retrieve == Some(false) {
```

Comment:
This and `CollectionQueryGroupsRequest` `check_customt` function seems duplicated code. Can we extract it into a separate function?

Reply:
it is duplicated indeed

Reply:
https://github.com/qdrant/qdrant/pull/6317/commits/70cad84e1329500a82f2d62046cc5ad07707040e